Projects/App/Sources/Presentation/MainHome/Subview/MainHomeDimView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  MainHomeDimView.swift
+//  App
+//
+//  Created by kokojong on 2023/02/12.
+//  Copyright © 2023 com.promise8. All rights reserved.
+//
+
+import UIKit
+
+final class MainHomeDimView: UIView {
+    
+    let addPromiseButton: UIButton = {
+        let button = UIButton()
+        button.backgroundColor = .wwwColor(.WWWBlack)
+        button.clipsToBounds = true
+        button.layer.cornerRadius = 46/2
+        button.setTitleColor(.wwwColor(.WWWWhite), for: .normal)
+        button.setTitle("새로운 약속 생성하기", for: .normal)
+        button.titleLabel?.font = .www(size: 14, family: .Bold)
+        button.contentEdgeInsets = UIEdgeInsets(top: 14, left: 14, bottom: 14, right: 14)
+        return button
```

Comment:
p2) 
웰컴투 iOS15...! 

deprecated 경고라 상관은 없지만 iOS 15 온김에 한번 사용해보는건 어때요? 
<img width="300" alt="스크린샷 2023-02-21 오후 5 34 38" src="https://user-images.githubusercontent.com/63157395/220291094-45483e8d-5cef-4ce7-ba91-b2abc15a04c8.png">


```suggestion
        let button = UIButton(type: .system)
        var config = UIButton.Configuration.filled()
        config.titleAlignment = .center
        config.baseBackgroundColor = .wwwColor(.WWWBlack)
        var attr = AttributedString.init("새로운 약속 생성하기")
        attr.font = .www(size: 14, family: .Bold)
        config.attributedTitle = attr
        config.cornerStyle = .capsule
        config.contentInsets = NSDirectionalEdgeInsets(top: 14, leading: 14, bottom: 14, trailing: 14)
        button.configuration = config
        return button
```

Reply:
아맞당 이걸로 하면 버튼 누르는 효과도 생김! 

https://user-images.githubusercontent.com/63157395/220293899-e3f2c2b9-0d9d-4ae5-986b-3449b41b43bd.mp4

Reply:
@chaneeii 
iOS 12에서 살던 나...저렇게 하는지 몰라따...
9374de3ebd478423e2e7901d4e595c1a86895fbb 에 반영 완료!